### PR TITLE
Framework

### PR DESCRIPTION
**Pull request by zhuchini**

### CODE REVIEW

**Comment by zhuchini** [comment]
The reason why I worked on the project solo is because my partner was absent the whole week and I was not able to communicate with them.

File: framework.css [hunk 1]
```diff
@@ -0,0 +1,1372 @@
...
+}
+
+button {
+  border-radius: 0;
+}
+
+.row {
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
```
Comment (samgaco) [inline framework.css]: :100:

File: framework.css [hunk 2]
```diff
@@ -0,0 +1,1372 @@
...
+
+  .cards-learning {
+    display: flex;
+  }
+}
+
+@media (max-width: 1024px) {
+  nav .navbar-brand {
+    padding-left: 0;
+  }
```
Comment (samgaco) [inline framework.css]: :white_check_mark:

**Comment by zhuchini** [comment]
Created a media range query for medium screens.

**Pull request merged**